gratipay/project_review_process.py

```diff
@@ -31,12 +42,31 @@ def create_issue(self, *teams):
             title = "{} and {}".format(*[t.name for t in teams])
         else:
             title = "{} and {} other projects".format(teams[0].name, nteams-1)
+
         body = []
+        team_ids = []
+        owner_usernames = set()
         for team in teams:
+            team_ids.append(team.id)
+            owner_usernames.add(team.owner)
             body.append('https://gratipay.com{}'.format(team.url_path))
+        assert len(owner_usernames) == 1, owner_usernames
         body.extend(['', '(This application will remain open for at least a week.)'])
         data = json.dumps({'title': title, 'body': '\n'.join(body)})
-        return self._poster.post(data)
+        review_url = self._poster.post(data)
+
+        self.db.run("UPDATE teams SET review_url=%s WHERE id = ANY(%s)", (review_url, team_ids))
+        [team.set_attributes(review_url=review_url) for team in teams]
+
+        owner = Participant.from_username(owner_usernames.pop())
```

Comment:
Might be better to use IDs instead of usernames for this (asserting we only have one unique value in the list + constructing a participant object)?